src/components/AutoComplete/AutoComplete.stories.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-alert */
```

Comment:
Forget it here?

Reply:
Yeah guess this was an issue the foreman lint config but not a problem here. Good catch!